README.md

```diff
@@ -415,6 +415,12 @@ z.B.
 │       └── Button.tsx
 ```
 
+#### Weitere
+
+| Strategie                                                                                   |
+|---------------------------------------------------------------------------------------------|
+| [Query Resultate einheitlich verarbeiten](doc/loesungsstrategien/frontend/queryboundary.md) |
```

Comment:
Wieso lagerst du die jetzt aus? Hatte das eigentlich beabsichtigter weise alls in die Readme aufgenommen und nicht ausgelagert